src/platform/Linux/ConnectivityManagerImpl.h

```diff
@@ -266,6 +266,8 @@ class ConnectivityManagerImpl final : public ConnectivityManager,
     uint16_t mApFreq;
     CHIP_ERROR _WiFiPAFPublish(WiFiPAFAdvertiseParam & args);
     CHIP_ERROR _WiFiPAFCancelPublish(uint32_t PublishId);
+    bool _WiFiPAFResourceAvailable() { return mPafChnlAvailable; };
+    bool mPafChnlAvailable = true;
```

Comment:
I would suggest not using abbreviations - prefer readability to short text.

Why is this `Chnl` when the value is returned by ResourceAvailable? what does this member mean and why does it start off as true?

Reply:
1. I will use the full name.
2. This function, WiFiPAFResourceAvailable(), is expected to blocking the upper layer to send the PAF packets if the resource is unavailable. Channel unavailable is a reason for now. If more items are found, we should add them here so that the interface name is not needed to be changed.
3. PAF channel should be always available, except the condition that the device may switch to others for some other tasks. That's why it's true by default